superset-frontend/src/components/PageHeaderWithActions/index.tsx

```diff
@@ -152,6 +158,8 @@ export const PageHeaderWithActions = ({
               css={menuTriggerStyles}
               buttonStyle="tertiary"
               aria-label={t('Menu actions trigger')}
+              tooltip={tooltipProps?.text ?? undefined}
+              placement={tooltipProps?.placement ?? undefined}
```

Comment:
```suggestion
              tooltip={tooltipProps?.text}
              placement={tooltipProps?.placement}
```

Reply:
Fixed in [`this commit`](https://github.com/apache/superset/pull/21189/commits/890af00638acaa42d9f7d35bd86f3ad573739122).